setup.py

```diff
@@ -57,6 +57,9 @@ def readme():
             'google-cloud-bigquery~=3.0',
             'db-dtypes==1.0.5',
         ],
+        'chroma': [
+            'chroma>=0.4.17',
```

Comment:
Shouldn't these be chromadb instead of chroma? 

CC
@wangxiaoyou1993

Reply:
oh, yeah, that's a good catch. will fix it.